doc/whats_new/v0.24.rst

```diff
@@ -235,6 +235,10 @@ Changelog
   will be removed in 0.28. :pr:`17662` by
   :user:`Joshua Newton <joshuacwnewton>`.
 
+- |Fix| Fix :issue:`10493`. MemoryError exception was raised when 
+  calling barycenter_weights with large inputs. 
```

Comment:
Change log entries should refer to the affected *public* API items. `barycenter_weights` is private. Could you try give some sense of how this change affects the user?